spec/02-integration/09-hybrid_mode/08-lazy_export.lua

```diff
@@ -0,0 +1,139 @@
+local helpers = require "spec.helpers"
+
+local admin_client
+
+local function cp(strategy)
+  assert(helpers.start_kong({
+    role = "control_plane",
+    cluster_cert = "spec/fixtures/ocsp_certs/kong_clustering.crt",
+    cluster_cert_key = "spec/fixtures/ocsp_certs/kong_clustering.key",
+    database = strategy,
+    cluster_listen = "127.0.0.1:9005",
+    nginx_conf = "spec/fixtures/custom_nginx.template",
+    -- additional attributes for PKI:
+    cluster_mtls = "pki",
+    cluster_ca_cert = "spec/fixtures/ocsp_certs/ca.crt",
+  }))
+  admin_client = assert(helpers.admin_client())
+end
+
+local n = 0
+local function touch_config()
+  n = n + 1
+  admin_client:send({
```

Comment:
The `:send()` method on our test http client doesn't do any error-checking. Adding an assertion or two here would help to make this test suite more debug-able.